### ISSUE TITLE
On rightclick on stairs (works on only 1 side?)

### ISSUE THREAD

**Issue by bolleboor** [SkriptLang/Skript#2055]
### Description

I tried to create a system where you can sit on a stair.
It works fine BUT, the stair must be facing a specific way or it doesn't work.
For example, mine needs to be facing to the west and becomes sittable.
This bug activates every time you use "on rightclick on stair" or any specific stair.

(This has NOTHING to do with ANY addons, this bug also occours when you are just trying  to send a basic message)
### Steps to Reproduce

on rightclick on stair:
    --Type any message 
### Errors / Screenshots

There are not errors in console.
Here's a gif: https://gyazo.com/a47a47dd7b0f685acb6a6e243811c3c3

### Server Information
* **Server version/platform:**  Paperspigot 1.12.2     
* **Skript version:**2.3.6 (Latest)

**Comment by Moderocky** [comment]
Have you tried `on rightclick on any (type) stair:`?
I'm not certain this works, but with a similar problem in the past specifying `any` before it worked for me.

If that doesn't help, as a short-term workaround, you can put the clicked block into a string and check what the common part of the string is for all stair orientations, for instance, the directions might be `north-facing brick stair` or `east-facing brick stair` or whatever, but they will all contain something like "brick stair" (if it is a brick stair.)

Then you might use something along the lines of:
```
on rightclick:
    event-block exists
    "%event-block%" contains "brick stair"
    #do your code
```

I've not tested that specifically, but it's similar to what I try when item/block comparisons don't work for me.

**Comment by TheBentoBox** [comment]
This should be resolved with the merge of #2030.
